src/components/pages/register.yml

```diff
@@ -7,9 +7,16 @@ termsAndConditionsConfirmation:
 register:
   en: Register
   de: Registrieren
-registrationInProgress:
-  en: We have sent you an email. There you will find the next steps in order to complete the registration.
-  de: Wir haben Ihnen eine E-Mail geschickt. Dort finden Sie die nächsten Schritte, um Ihre Registrierung abzuschließen.
-redirectMessage:
-  en: You will be redirected to the <0>logon page</0> in {seconds, plural, =1 {# second} other {# seconds}}.
-  de: Sie werden in {seconds, plural, =1 {# Sekunde} other {# Sekunden}} auf die <0>Anmeldeseite</0> weitergeleitet.
+registrationConfirmationHeader:
+  en: One more step remaining…
+  de: Ein Schritt fehlt noch…
```

Comment:
A space between "remaining" / "noch" and "..." would make it a bit more readable I feel.

Reply:
Done, looks indeed better, thanks!